### DISCUSSION TITLE
Is it possible to inject one Remix app into another?

### DISCUSSION THREAD

**Discussion by Foprta** [tiny-frontend/tiny-client#12] [Q&A] [answered]
In your Remix example you are using Remix as host and basic React component as injection.

Is it possible to build Remix app the way it can be used as injection?

I am wondering how to combine some applications with their own routes and lazy loading.

**Comment by adrienbaron** [comment, marked as answer]
Is what you mean to inject a Remix app in another?
If so then tiny client isn't the right tool for that, it's purposely built to include small bits of JS in another app at runtime.
However I've built an example for remix federation that does what you want, it requires forking Remix though and it's just a demo, so use at your own risk 😅:

https://github.com/remix-run/remix/discussions/1989

**Reply by Foprta** [reply]
Very interesting. Thanks!
